bin/state

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
 # Run Terraform commands on the create_state_bucket project in the Hashicorp
 # docker container, passing our environment variables
 DOCKER_USER="$(id -u):$(id -g)" \
     docker-compose run --rm \
-        -v "${PWD}/src/aws/create_state_bucket:/app"\
-        terraform "$@"
+    -v "${PWD}/src/aws/create_state_bucket:/app"\
+    terraform "$@"
```

Comment:
I find previous indentation more readable.

Reply:
completely... this was not intentional.